File: Modules/SEDO/EMS/SIE.Web.EMS/SpareParts/Applys/Commands/SaveAppCommand.cs
using SIE.Domain;
using SIE.EMS.SpareParts.Applys;
using SIE.EMS.SpareParts.Applys.Controllers;
using SIE.Web.Command;

namespace SIE.Web.EMS.SpareParts.Applys.Commands
{
    /// <summary>
    /// 保存
    /// </summary>
    [JsCommand("SIE.Web.EMS.SpareParts.Applys.Commands.SaveAppCommand")]
    public class SaveAppCommand : FormSaveCommand
    {
        /// <summary>
        /// 保存
        /// </summary>
        /// <param name="entity"></param>
        protected override void DoSave(Entity entity)
        {
            var spa = entity as SparePartApp;
           
            //保存
            RT.Service.Resolve<SparePartAppController>().SparePartAppSave(spa);
        }
    }
}

File: Modules/SEDO/EMS/SIE.Web.EMS/SpareParts/Applys/Commands/SaveAppCommand.js
SIE.defineCommand('SIE.Web.EMS.SpareParts.Applys.Commands.SaveAppCommand', {
    extend: 'SIE.cmd.FormSave',
    meta: { text: "保存", group: "edit", iconCls: "icon-SaveEntity icon-blue" },
    canExecute: function (view) {
        return view.getData() && view.getData().isDirty();
    }
});